rfcs/protocols/data-sharing/fabric.md

```diff
@@ -8,7 +8,50 @@
 - RFC: 02-002
 - Authors: Venkatraman Ramakrishna, Sandeep Nishad, Krishnasuri Narayanam, Dhinakaran Vinayagamurthy
 - Status: Proposed
-- Since: 01-Dec-2020
+- Since: 02-Jan-2023
 
 ## Summary
 
+This document specifies the Hyperledger Fabric implementation of modules, and application adaptation guidelines, for the asset transfer protocol.
```

Comment:
Why does data-sharing rfc says this is for asset transfer? Even the next line is also about asset transfer

Reply:
Typo. This file was initially copied from the corresponding asset transfer RFC.